packages/block-library/src/social-link/index.php

```diff
@@ -92,6 +92,24 @@ function register_block_core_social_link() {
 			)
 		);
 	}
+
+	register_block_type(
+		'core/social-link',
+		array(
```

Comment:
With `block.json` in place, the following is possible:

https://github.com/WordPress/gutenberg/blob/2c587e5e252852870c087c1d9874d8867aa53c6d/packages/block-library/src/shortcode/index.php#L24-L34

It helps to avoid syncing attributes between PHP and JS. WordPress core has everything in place to handle it.

Reply:
True, thanks!